pages/faq/troubleshooting/faq.md

```diff
@@ -24,6 +24,7 @@ title: FAQs
 * [How do you secure your own “cloud” to prevent malicious attack which may allow attacker to break-in our systems?](#how-do-you-secure-your-own-cloud-to-prevent-malicious-attack-which-may-allow-attacker-to-break-in-our-systems)
 * [What does it mean when a device type is discontinued?](#what-does-it-mean-when-a-device-type-is-discontinued)
 * [I have a device that is not on the supported devices list. Can it run on {{ $names.company.lower }}?](#i-have-a-device-that-is-not-on-the-supported-devices-list-can-it-run-on-{{ $names.company.lower }})
+* [What to keep in mind when choosing supply units?](#what-to-keep-in-mind-when-choosing-power-supply-units)
```

Comment:
Should "choosing supply units" be "choosing power supply units" ?!

Reply:
Good find. Missed that bit, thanks!